core/src/commonMain/kotlin/com/sourcepoint/mobile_core/models/consents/GlobalCmpConsent.kt

```diff
@@ -0,0 +1,19 @@
+package com.sourcepoint.mobile_core.models.consents
+
+import com.sourcepoint.mobile_core.models.consents.USNatConsent.USNatUserConsents
+import com.sourcepoint.mobile_core.utils.inOneYear
+import com.sourcepoint.mobile_core.utils.now
+import kotlinx.datetime.Instant
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class GlobalCmpConsent (
+    val applies: Boolean = false,
+    val categories: List<String> = emptyList(),
+    val consentStatus: ConsentStatus = ConsentStatus(),
+    val dateCreated: Instant = now(),
+    val expirationDate: Instant = dateCreated.inOneYear(),
+    val gpcEnabled: Boolean? = null,
+    val uuid: String? = null,
+    val userConsents: USNatUserConsents = USNatUserConsents()
```

Comment:
Maybe we should create an empty class for the global cmp user consents. Something like `GlobalCMPUserConsents`? Not sure if it's an overkill. Or, if we notice it's the exact same data as `USNatUserConsents`, we could rename that to be more generic.

Let's keep the way it is for now, until we have the data coming from wrapper.